simplecpp.cpp

```diff
@@ -1737,16 +1739,10 @@ namespace {
         }
     }
 
+    const char * const altopData[] = {"and","or","bitand","bitor","not","not_eq","xor"};
```

Comment:
I don't have a strong opinion about this one. whatever astyle outputs is proper coding style.